docs/start/smesher/post_1n.md

```diff
@@ -73,41 +73,35 @@ Run it with the following command:
 
 Where: `source_path` is the path to the node data you want to merge from and `target_path` is the path to the node data you want to merge to.
 
-:::note
-
-It is possible to merge nodes by hand too
-
-- Locate the `identity.key` files within the PoST data directories of each node.
-- Copy these files to the `./node_data/identities` directory on the node you're consolidating to.
-- Rename the key files respectively for easy identification of each identity.
-- Run: `sqlite3 target_node.sql` where `target_node.sql` is the database file of the node you're consolidating to.
-
-```attach '<source_path.sql>' as srcDB;
-BEGIN;
-insert into initial_post select * from srcDB.initial_post;
-insert into challenge select * from srcDB.challenge;
-insert into poet_registration select * from srcDB.poet_registration;
-insert into nipost select * from srcDB.nipost;
-COMMIT;
-detach srcDB;
-```
-
-It is however recommended only for advanced users as it does not cover all the corner cases and may lead to data corruption.
-
-:::
+> [!NOTE]
```

Comment:
This is GH syntax. On that site we are using docusaurus which uses the syntax that was there :)